tracer/build/_build/Build.Steps.cs

```diff
@@ -128,6 +128,9 @@ partial class Build
             ? new[] { TargetFramework.NET462, TargetFramework.NETCOREAPP2_1, TargetFramework.NETCOREAPP3_0, TargetFramework.NETCOREAPP3_1, TargetFramework.NET5_0, TargetFramework.NET6_0, TargetFramework.NET7_0, }
             : new[] { TargetFramework.NET462, TargetFramework.NETCOREAPP2_1, TargetFramework.NETCOREAPP3_1, TargetFramework.NET7_0, };
 
+    TargetFramework[] TestingFrameworksDebugger =>
+        TargetFramework.GetFrameworks(except: new[] { TargetFramework.NET461, TargetFramework.NETSTANDARD2_0 });
+
```

Comment:
nit: as this is only used in one place, I would inline it into `Build.VariableGenerations`

Reply:
Sounds reasonable.
Refactored at 1cffeb5777c44f47d709e3a78506345f7c5322bb

Reply:
oh, no, that's not what I meant 😄 I meant `TestingFrameworksDebugger` doesn't need to be a property (and exposed everywhere) But meh, it's fine. `TestingFrameworks` is also used in Build.Steps (or at least it _should_ be, see #3542 🙄)